charts/kserve-resources/templates/configmap.yaml

```diff
@@ -697,4 +697,6 @@ data:
       "scrapeInterval": "5s",
       "metricReceiverEndpoint": "keda-otel-scaler.keda.svc:4317",
       "metricScalerEndpoint": "keda-otel-scaler.keda.svc:4318"
+      "scaleUpStabilizationWindowSeconds": "0",
+      "scaleDownStabilizationWindowSeconds": "300"
```

Comment:
this section is for otel collector, these two settings can be moved out to the autoscaling section.